server.js

```diff
@@ -24,5 +41,16 @@ app.get("/about", (req, res) => {
   res.render("pages/about")
 })
 
-app.listen(8080)
-console.log("Server listening on port 8080")
+const start = async () => {
+  try {
+    const conn = await connectDB(process.env.MONGO_URI)
+    console.log(`MongoDB: ${conn.connection.host}`.green.underline)
```

Comment:
Same thing about required env vars, although in this case the MongoDB library has your back: notice that it does fail early and loudly if it gets an empty value, as all things should!